`brotli` feature without `gzip` fails to compile

I hit this with `v0.14.2` because that's what `arrow2` latest is using:

```
   Compiling parquet2 v0.14.2
error[E0433]: failed to resolve: use of undeclared crate or module `flate2`
   --> /Users/kyle/.cargo/registry/src/github.com-1ecc6299db9ec823/parquet2-0.14.2/src/parquet_bridge.rs:187:28
    |
187 | impl From<BrotliLevel> for flate2::Compression {
    |                            ^^^^^^ use of undeclared crate or module `flate2`
```

But it seems this is still an issue on latest `main`. In `Cargo.toml` the `brotli` feature doesn't seem to require `flate2`, but this `impl` is still defined
https://github.com/jorgecarleitao/parquet2/blob/4cbce012cfe8477bf231caf2fd683871262ba0d4/src/parquet_bridge.rs#L186-L191

--- a/src/parquet_bridge.rs
+++ b/src/parquet_bridge.rs
@@ -183,13 +183,6 @@ impl BrotliLevel {
     }
 }
 
-#[cfg(feature = "brotli")]
-impl From<BrotliLevel> for flate2::Compression {
-    fn from(level: BrotliLevel) -> Self {
-        Self::new(level.compression_level() as u32)
-    }
-}
-
 /// Represents a valid gzip compression level.
 #[derive(Debug, Eq, PartialEq, Hash, Clone, Copy)]
 pub struct GzipLevel(u8);

